build.gradle

```diff
@@ -69,6 +69,18 @@ task installRealmJava(type:Task) {
     description = 'Install the Realm library and Gradle plugin into mavenLocal()'
 }
 
+task installRobolectric(type:GradleBuild) {
+    description = 'Install the Robolectric library into example'
```

Comment:
@zaki50 Since I modified `realm-android` plugin and added `realm-robolectric`, I recommend using `./gradlew installRealmJava` to install both plugins.